### PR TITLE
[CI] Updated core workflow for E2E

### PR DESCRIPTION
**Pull request by Akshit42-hue**
Signed-off-by: Axit Patel <patelakshit2025@gmail.com>

**Description**

This PR is a part of  https://github.com/meshery/meshery/issues/5529

**Notes for Reviewers**
Ports the Cilium to the new core e2e workflow

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/workflows/e2etest.yaml [hunk 1]
```diff
@@ -108,11 +109,11 @@ jobs:
...
-              traefik-mesh-controller: $(jq '.pods_status."traefik-mesh-controller"' data.json)
-              traefik-mesh-proxy: $(jq '.pods_status."traefik-mesh-proxy"' data.json)
-              grafana-core:  $(jq '.pods_status."grafana-core"' data.json)
-              jaeger: $(jq '.pods_status."jaeger"' data.json)
-              prometheus-core: $(jq '.pods_status."prometheus-core"' data.json) 
+              traefik-mesh-controller: $(jq '.resources_status."pod/traefik-mesh-controller"' data.json)
+              traefik-mesh-proxy: $(jq '.resources_status."pod/traefik-mesh-proxy"' data.json)
+              grafana-core:  $(jq '.resources_status."pod/grafana-core"' data.json)
+              jaeger: $(jq '.resources_status."pod/jaeger"' data.json)
+              prometheus-core: $(jq '.resources_status."pod/prometheus-core"' data.json)
```
[the remark this answers is not in the document]
Comment (Akshit42-hue) [reply .github/workflows/e2etest.yaml:116]: Yes, looking into it :)

**Comment by Akshit42-hue** [comment]
Required changes have been done @gr455

**Comment by gr455** [comment]
> Signed-off-by: Axit Patel [patelakshit2025@gmail.com](mailto:patelakshit2025@gmail.com)
> 
> **Description**
> 
> This PR fixes [meshery/meshery#5529](https://github.com/meshery/meshery/issues/5529)
> 
> **Notes for Reviewers** Ports the Cilium to the new core e2e workflow
> 
> **[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
> 
> * [x]  Yes, I signed my commits.

@Akshit42-hue Can you just change 'This PR fixes' to 'This PR is a part of' so that it doesn't close the issue on merge

**Review by gr455 — APPROVED** [review]
Looks good to me!
@Revolyssup Ready to merge

**Review by Revolyssup — APPROVED** [review]

**Pull request merged**